app/views/paye/paye_taxs_main.scala.html

```diff
@@ -37,7 +37,11 @@
 @(viewModel: PayeAtsMain)(implicit request: PayeAuthenticatedRequest[_], messages: Messages, lang: Lang, formPartialRetriever: FormPartialRetriever)
 
 @headerContent = {
-
+@if(lang.language == "cy"){
+<a id="english-switch" href=@controllers.routes.TaxsLanguageController.switchLanguage("en") data-journey-click="Language:Select:English">@Html(Messages("generic.english"))</a>
```

Comment:
Wrong way round

Reply:
If selected language is welsh then we need to show english switch otherwise it always has to be welsh.